cmd/main.go

```diff
@@ -14,9 +14,13 @@ import (
 // CommitString is the commit used to build the server
 var CommitString string
 
+const (
+	authorizedKeysPath = "/var/okteto/remote/authorized_keys"
+	shell              = "bash"
```

Comment:
Can we make the `bash` installation optional?
If we can find a portable "bash" binary it would be ideal, but if not, ssh will only work with root images

Reply:
It would be a separate PR

Reply:
I was looking at the code, I think we can remove that dependency from here, and just require `sh`